crates/bitwarden-crypto/src/rsa.rs

```diff
@@ -41,6 +41,9 @@ pub(crate) fn make_key_pair(key: &SymmetricCryptoKey) -> Result<RsaKeyPair> {
         SymmetricCryptoKey::Aes256CbcHmacKey(key) => {
             EncString::encrypt_aes256_hmac(pkcs.as_bytes(), key)
         }
+        SymmetricCryptoKey::XChaCha20Poly1305Key(_) => Err(CryptoError::OperationNotSupported(
```

Comment:
Perhaps comment that we do not intend to support chacha-wrapped rsa keys?

We don't really want someone coming along and realizing they could do this trivially

Reply:
We do! However, that is a follow-up PR once we have content-types. I'm not sure whether these should be cosekeys or just PKCS#8 yet. Currently I favor the latter, and then just migrate to HPKE keys using a better algorithm (DH-KEM).